templates/openstack/README.md

```diff
@@ -0,0 +1,31 @@
+Monitoring Openstack
+====================
+
+Monitoring Openstack environment using the Openstack Command Line client.
+
+* Images
+* Ports
+* Resource Limits
+* Server List
+* Resource Usage
+* Volume List
+* Zone Availability
+
+The monitoring requires the Openstack CLI to be installed on the (LINUX) server where the Patrol Agent and the Monitoring Studio KM is installed.
+
+https://docs.openstack.org/newton/user-guide/common/cli-install-openstack-command-line-clients.html
+
+Set up
+------
+
+* Create a user account for monitoring.
+
+* Set the Environment Variables: https://docs.openstack.org/newton/user-guide/common/cli-set-environment-variables-using-openstack-rc.html
+ An Openstack RC file is embedded in the template. Edit the below lines in the embedded "openrc-demo-monitor.sh" with appropriate values.
```

Comment:
You need to add an extra line break here (before "An Openstack RC"). Otherwise, it's concatenated as one line with the previous link.

Reply:
fixed.